setup.py

```diff
@@ -40,9 +40,7 @@ def read(rel_path):
     include_package_data=True,
     # See http://www.sphinx-doc.org/en/stable/theming.html#distribute-your-theme-as-a-python-package
     entry_points={"sphinx.html_themes": ["pyansys_sphinx_theme = pyansys_sphinx_theme"]},
-    install_requires=["sphinx",
-                      "pydata-sphinx-theme==0.6.3",
```

Comment:
You still need the theme.

Reply:
I added it to the requirements now. I realized later.